cmd/katalyst-agent/app/options/sysadvisor/sysadvisor_base.go

```diff
@@ -48,6 +49,7 @@ func NewGenericSysAdvisorOptions() *GenericSysAdvisorOptions {
 			types.AdvisorPluginNameMetaCache,
 			types.AdvisorPluginNameMetricEmitter,
 			types.AdvisorPluginNameInference,
+			types.AdvisorPluginNamePowerAware,
```

Comment:
disable power plugin by default

Reply:
will remove power plugin from the default enabled list